### PR TITLE
Add unit test for FtpAvailabilityChecker

### PR DESCRIPTION
**Pull request by KarolJastrzebski**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/sendletter/services/FtpAvailabilityCheckerTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.junit.Test;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FtpAvailabilityCheckerTest {
+
+    private static AbstractBooleanAssert<?> assertThatFtpCheck(String fromHour, String toHour, String now) {
```
Comment (lgonczar) [inline src/test/java/uk/gov/hmcts/reform/sendletter/services/FtpAvailabilityCheckerTest.java]: It would be best to have this at the bottom, after test methods.

File: src/test/java/uk/gov/hmcts/reform/sendletter/services/FtpAvailabilityCheckerTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        assertThatFtpCheck("16:00", "17:00", "15:00").isTrue();
+    }
+
+    @Test
+    public void ftp_is_available_after_downtime() {
+        assertThatFtpCheck("08:00", "09:00", "10:00").isTrue();
+    }
+
+    @Test
+    public void ftp_is_unavailable_during_downtime() {
```
Comment (lgonczar) [inline src/test/java/uk/gov/hmcts/reform/sendletter/services/FtpAvailabilityCheckerTest.java]: Please add edge cases from the existing tests (consumer).

File: src/test/java/uk/gov/hmcts/reform/sendletter/services/FtpAvailabilityCheckerTest.java (hunk 1 above)
Comment (KarolJastrzebski) [reply src/test/java/uk/gov/hmcts/reform/sendletter/services/FtpAvailabilityCheckerTest.java]: Although private, it is a static method and should be easier to spot at the beginning of the class. I would keep it where it is, but could move if you really, really wanted. :)
